http_client/views/http_client_auth_views.xml

```diff
@@ -86,6 +86,13 @@
                             />
                             <field name="access_token" invisible="1"/>
                             <field name="refresh_token" invisible="1"/>
+                            <div colspan="2">
+                                <p invisible="auth_method != 'api_key'">
```

Comment:
Will be good enough for now. Later I might implement clear interface for description, so you would provide it in code directly and view would show related description of each method (if it has any).